acceptance/indy_acceptance/indy-tests/libraries/common.py

```diff
@@ -0,0 +1,159 @@
+"""
+Created on Nov 13, 2017
+
+@author: khoi.ngo
+
+Containing all functions that is common among test scenarios.
+"""
+
+import asyncio
+import json
+from indy import wallet, pool, ledger
+from .constant import Colors, Constant, Message
+
+
+class Common:
+    """
+    Wrapper common function for test scenario.
+    """
+
+    @staticmethod
+    async def prepare_pool_and_wallet(pool_name, wallet_name, pool_genesis_txn_file):
+        """
+        Prepare pool and wallet to use in a test case.
+
+        :param pool_name: Name of the pool ledger configuration.
+        :param wallet_name: Name of the wallet.
+        :param pool_genesis_txn_file: The path of the pool_genesis_transaction file.
+        :return: The pool handle and the wallet handle were created.
+        """
+        pool_handle = await Common().create_and_open_pool(pool_name, pool_genesis_txn_file)
```

Comment:
There is no need to instantiate a class to access its static member. This line might be corrected as:
`pool_handle = await Common.create_and_open_pool(pool_name, pool_genesis_txn_file)`

Reply:
Updated. Please refer to the latest code.